deploy/helm/flightctl/charts/keycloak/values.yaml

```
@@ -0,0 +1,37 @@
+keycloak:
+  rootUrl: https://flightctl.example.com:443
+  redirectUris:
+    - https://flightctl.example.com:443/*
+  authority: http://keycloak:8080/realms/flightctl
+  clientid: flightctl-ui
+  redirect: https://ui.flightctl.example.com:443
+  secret: CHANGE_ME
+  namespace: flightctl
```

we prepare flightctl-external and flightctl-internal namespaces in kind.

It's failing because of that
```suggestion
  namespace: flightctl-external
```